### PR TITLE
feat: add install and migration documentation

### PR DESCRIPTION
**Pull request by vishal-chdhry**

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+You can install reports server in any namespace. This example uses `reports-server` as the namespace:
+
+```bash
+kubectl create namespace reports-server
+```
+
+Install the reports-server chart:
+```bash
+helm install reports-server --namespace reports-server reports-server/reports-server --devel
```
Comment (realshuting) [inline docs/INSTALL.md:44]: `--devel` is for pre-release only.

Comment (vishal-chdhry) [reply docs/INSTALL.md:44]: yes but I could not install the chart without the flag at the moment

Comment (realshuting) [reply docs/INSTALL.md:44]: Yes because we are on alpha versions. For GA release the user doesn't need this flag.

Comment (realshuting) [reply docs/INSTALL.md:44]: @vishal-chdhry ?

Comment (vishal-chdhry) [reply docs/INSTALL.md]: @realshuting updated

**Review by realshuting — APPROVED** [review]

File: docs/INSTALL.md (hunk 1 above)
Comment (realshuting) [reply docs/INSTALL.md]: thx!

**Pull request merged**